.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
   - sudo sysctl -w vm.max_map_count=262144
   # Build images
   - docker login -u aschen -p $DOCKER_PASSWORD
```

Comment:
why are you logging on DockerHub as Adrien?

Reply:
Oups, this is the master's branch file i haven't change it

Reply:
ping @alexandrebouthinon :)

Reply:
@xbill82 : Let's merge this PR since this setup was there for a while. I'll take care of it soon :slightly_smiling_face:

Reply:
We already had this conversation: https://github.com/kuzzleio/kuzzle-how-to/pull/15#discussion_r184107812